### DISCUSSION TITLE
Component HTML with attributes data-gjs-*

### DISCUSSION THREAD

**Discussion by iuzhakov** [GrapesJS/grapesjs#5730] [Q&A] [answered]
Hello!

I have a plugin with components and I want to add data attributes to some elements

Example:
`function myPlugin(editor) {
  editor.Blocks.add('my-first-block', {
    label: 'Simple block',
    content: '<div class="my-block"><div>Block 1</div><div data-gjs-copyable="false" data-gjs-removable="false">Block 2</div></div>',
  });
}`

I want to hide toolbar buttons - copy and remove - and it works fine

When I try to get component HTML - component.toHTML({withProps: true}) - there is no data attributes `data-gjs-copyable="false" data-gjs-removable="false"` - only `data-gjs-type="my-first-block"`

Does anybody know how to get all data attributes (copyable, removable, draggable....)?

Thank you!

**Comment by artf** [comment]
Looks like there is an issue with booleans

**Comment by artf** [comment, marked as answer]
Fixed for the next release https://github.com/GrapesJS/grapesjs/commit/d406685708a018ec733f84cd7974af6ca7543fe7

**Reply by iuzhakov** [reply]
It works! Thank you!
